docker-alfresco/pom.xml

```diff
@@ -156,13 +155,20 @@
                                 </artifactItem>
                                 <artifactItem>
                                     <groupId>org.alfresco.integrations</groupId>
-                                    <artifactId>alfresco-googledocs-repo</artifactId>
+                                    <artifactId>alfresco-googledocs-repo-community</artifactId>
                                     <version>${alfresco.googledocs.version}</version>
-                                    <classifier>community</classifier>
                                     <type>amp</type>
                                     <overWrite>false</overWrite>
                                     <outputDirectory>${project.build.directory}/amps</outputDirectory>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>org.alfresco.integrations</groupId>
+                                    <artifactId>alfresco-googledocs-share-community</artifactId>
```

Comment:
there is no share community or share enterprise version, therefore, the googledocs -share amp for community does not fit with the current model.
There is only one share docker image currently build and published - used in both enterprise and community ACS releases. 
Of course this could change, if required: Let the Share team know